sip/sip-005-blocks-and-transactions.md

```diff
@@ -1229,13 +1229,15 @@ A non-fungible token owned by an account is encoded as follows:
 * Key: the string `"vm::"`, the fully-qualified contract identifier, the string `"::4::"`,
   the name of the token as defined in its Clarity contract, the string `"::"`,
 and the serialized Clarity value that represents the token.
-* Value: a serialized Clarity standard principal
+* Value: a serialized Clarity Principal (either a Standard Principal or a Contract Principal)
 
 Example: `"vm::SP13N5TE1FBBGRZD1FCM49QDGN32WAXM2E5F8WT2G.example-contract::4::example-nft::\x02\x00\x00\x00\x0b\x68\x65\x6c\x6c\x6f\x20\x77\x6f\x72\x6c\x64"` 
 refers to the non-fungible token `"hello world"` (which has type `buff` and is
 comprised of 11 bytes), defined in Clarity contract `SP13N5TE1FBBGRZD1FCM49QDGN32WAXM2E5F8WT2G.example-contract`
 as a type of non-fungible token `example-nft`.
 
+A Principal Address is either a 32check-encoded address in the case of standard principal, or a c32check-encoded address, followed by an ASCII period `.`0, and an ASCII-encoded string for a contract principal.
```

Comment:
Typo -- should be `c32check-encoded`.  Also, it looks like you have an extra 0 in "`.`0"

Reply:
thanks for catching this, embarrassing...  fixed in 30875ff